src/django/api/services/facilities_download_service.py

```
@@ -63,47 +73,27 @@ def get_download_limit(request):
-        )
-
-        if has_exhausted_limit:
+        if allowed == 0:
             raise ValidationError(
-                'You have reached your annual limit for facility record '
-                'downloads, including both free and paid. Additional '
-                'downloads will be available at the start of the next '
-                'calendar year.'
+                "You have reached your annual limit "
+                "for facility record downloads..."
             )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)